statick_tool/config.py

```diff
@@ -157,7 +157,13 @@ def get_tool_config(
         self, plugin: str, level: str, key: str, default: Optional[str] = None
     ) -> Optional[str]:
         """Get tool flags to use for a plugin at a certain level."""
-        return self.get_plugin_config("tool", plugin, level, key, default)
+        tool_flags = self.get_plugin_config("tool", plugin, level, key, default)
+        # Make sure the flags are on a single line string and remove double-quotes and
+        # whitespace that might have been added from yaml multi-line syntax.
+        if tool_flags is not None:
+            tool_flags = " ".join(tool_flags.split()).strip('"').strip()
```

Comment:
why is removing `"` now required?

Reply:
When we have all the flags on a single line and read them in there are no double quotes around the flags (when they are represented as a string in Statick code). The steps in `" ".join(tool_flags.split())` add the double quotes if the flags are specified using multi-line yaml syntax.

Reply:
I'm wondering why this part is needed `.strip('"')`

Reply:
That removes the leading and trailing `"` characters that are added when the flags are specified using multi-line yaml syntax. If the flags are specified on a single line then the `"` characters do not get added and the `.strip('"')` is a no-op.

Reply:
Note that the only tool that is sensitive to the leading/trailing `"` is `make`. That tool needs very specific formatting of flags that no other tools seem to care about.